coalib/parsing/LineParser.py

```diff
@@ -55,6 +56,19 @@ def parse(self, line):
         line, comment = self.__separate_by_first_occurrence(
             line,
             self.comment_separators)
+
+        # Comment starting with # should have whitespace after #
+        # Example: C# is the hybrid of C and C++.
+        if '#' in line:
+            start_index = line.index('#')
+            if start_index == 0 and line[1] != ' ':
+                logging.warning('The comment line does not have \
+                                 whitespace after #')
+            elif line[start_index - 1] != ' ' \
```

Comment:
Yes. You don't have to worry about it starting with ```#``` as it will be normal comment and we don't want to issue a warning for that.